test/src/test/java/org/corfudb/integration/CorfuReplicationReconfigurationIT.java

```diff
@@ -520,8 +524,12 @@ public void testStandbyStreaming() throws Exception {
 
             // Attach new listener for deltas (the same listener could be used) but simplifying the use of the latch
             CountDownLatch streamingStandbyDeltaCompletion = new CountDownLatch(totalEntries*2);
+
+            // The number of expected transactions to be received on the listener
+            CountDownLatch numTxObserved = new CountDownLatch(totalEntries*2);
+
```

Comment:
I think these 2 tx latch are supposed to be awaited?

Reply:
yes, good catch.  Added the awaits and also changed the variable names and comments to make it clearer